src/screens/Bridge/Exchange-CrossChain/CrossChain.tsx

```diff
@@ -148,25 +148,20 @@ function CrossChain({ fetchCrossChainTitle }: Props) {
 
   const txData = React.useMemo(() => {
     if (!buildTransactionData) return null;
-    const { approvalTransactionData, transactionData } = buildTransactionData;
-    if (!approvalTransactionData) return [transactionData];
-    return [approvalTransactionData, transactionData];
+    const { transactions } = buildTransactionData;
+    return transactions;
   }, [buildTransactionData]);
 
   const offer = React.useMemo(() => {
     if (!buildTransactionData) return null;
-    const {
-      provider,
-      estimate: { data, toAmount },
-    } = buildTransactionData.quote;
-    const { fromToken, toToken } = data;
+    const { toAmount, fromToken, toToken, gasCostUSD } = buildTransactionData.route;
 
     const decimalValue: any = `10e${toToken?.decimals - 1}`;
 
     const amount: any = parseInt(toAmount) / (decimalValue ?? 1);
 
     return {
-      provider: provider === 'lifi' ? 'Lifi' : provider,
+      provider: 'Lifi',
```

Comment:
We've hardcoded 'Lifi' here....

Reply:
Yes, `getAdvanceRoutesLiFi` does not provide Lifi provider name